monai/data/image_writer.py

```diff
@@ -373,11 +373,11 @@ class ITKWriter(ImageWriter):
     output_dtype: DtypeLike = None
     channel_dim: int | None
 
-    def __init__(self, output_dtype: DtypeLike = np.float32, affine_lps_to_ras: bool = True, **kwargs):
+    def __init__(self, output_dtype: DtypeLike = np.float32, affine_lps_to_ras: bool | None = None, **kwargs):
```

Comment:
Not sure why we need this default value change here? But I find another wired thing just using the data provided in this [issue](https://github.com/Project-MONAI/MONAI/issues/6136). The "nrrd" data after `SaveImage` will have affine with shape (5, 5), It looks like it assumes that the data becomes 4-dimensional, without excluding the channel dimension.

```
original_data = LoadImage(image_only=True)(original_file)
SaveImage(output_dir="./monai_modified_meta",
          output_ext = ".nrrd",
          writer= "ITKWriter",
          resample=False,
          separate_folder= False,
          squeeze_end_dims = False,
          output_dtype=itk.SI)(original_data)
print(original_data.meta["spatial_shape"]) # --> ([512 512   1])

save_data = LoadImage(image_only=True)(save_data_path)
print(save_data.meta["spatial_shape"]) # --> [512 512   1   1]
```

Reply:
thanks, that's a bug and this https://github.com/Project-MONAI/MONAI/commit/91d974f2e1654c391781b4204ee87635659100b3 fixes it. 

(`SaveImage` has some logic to handle the test case even if it's not channel first: https://github.com/Project-MONAI/MONAI/blob/778d0aa558b69a600e4db74e7d59490b7ca9370f/monai/transforms/io/array.py#L453-L454)